### ISSUE TITLE
Regression - Start in Tray broken

### ISSUE THREAD

**Issue by newhinton** [Zettlr/Zettlr#3669]
### Description

the `-m, --launch-minimized`-flags have no effect in the current develop branch.

https://github.com/Zettlr/Zettlr/blob/f94913d7eecb7ccc9a50f1044df67ee13fa58a86/source/app/service-providers/windows/index.ts#L255-L257

It seems the above snippet always calls the window to open, regardless of the flags set.

`this.maybeShowWindows()` will honor the settings and fix this issue, however i dont know why `this.showMainWindows()` was choosen in the first place, and where it may be required.

On a sidenote; 
https://github.com/Zettlr/Zettlr/blob/f94913d7eecb7ccc9a50f1044df67ee13fa58a86/source/app/service-providers/windows/index.ts#L294
There is a function-call for this, including a constant, so that both -m and --launch-minimized work. I will fix that if someone confirms that the proposed fix does not have sideeffects.

### Reproducing

- add -m or --launch-minimized flag to zettlr-executable
- start Zettlr
- See it open

### Zettlr Version

Right off the develop branch

### Installation Method

Other (please specify below)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Kubuntu 22.04

### Additional Information

Build fresh from dev

**Comment by nathanlesage** [comment]
Heya, the main problem is that after the refactor the document provider (which maintains the document tree, I.e. the data structure that contains the different split tabs) and the window provider are tightly coupled. When a new window is opened in the document provider, the window provider will open a corresponding window.

However, it should be fairly straight forward to re-implement the minimize thing. We only have to make sure that if the user then opens the windows that the window provider opens the corresponding windows. Basically it needs to open the correct amount of windows and provide the corresponding keys (uuid4) for every window. I can have a look, should be easy!

**Comment by nathanlesage** [comment]
Hey! First of all, apologies that it took me so long to fix that. I kept this issue constantly at the back of my mind ever since you opened it, but I didn't want to implement it back then because I was unsure as to how the (back then brand new) API shared between the DocumentManager and WindowProvider would evolve. By now, I think it's fairly stable, so I could implement it.

I also fixed the second issue you mentioned, regarding the launch minimized command.
